mailpoet/assets/css/src/mailpoet-templates.scss

```diff
@@ -0,0 +1,107 @@
+@import '../../../node_modules/@woocommerce/components/build-style/style';
+
+.mailpoet-templates-container {
+  display: flex;
+  flex-direction: column;
+  gap: 26px;
+  margin: auto;
+  max-width: 1072px;
+}
+
+.mailpoet-templates-heading {
+  align-items: center;
+  align-self: stretch;
+  display: flex;
+  justify-content: space-between;
+  padding: 16px 0;
+
+  h1 {
+    padding: 4px 0;
+    vertical-align: bottom;
+  }
+
+  .components-search-control {
+    width: 180px;
+  }
+
+  .components-base-control__field {
+    margin: 0;
+  }
+
+  .components-search-control input[type=search].components-search-control__input {
+    background: #fff;
```

Comment:
I don't know if it makes much of a difference, but I know we have variables for a lot of colors in `mailpoet/assets/css/src/settings/_colors.scss`, and it seems more idiomatic in the codebase to use those. Is there any reason you went with the hex values for colors that we have variables for?

Reply:
Good point. I failed to consider this. I updated the commit to use colors from `_colors.scss` whenever possible.